src/components/Tabber/Tabber.module.scss

```
@@ -0,0 +1,7 @@
+.social {
+  display: flex;
+
+  justify-content: start;
```

Aqui que está quebrando seu ci, eu acho.
```suggestion
  justify-content: flex-start;
```